keps/prod-readiness/sig-auth/3304.yaml

```diff
@@ -0,0 +1,3 @@
+kep-number: 3304
+stable:
```

Comment:
Straight to stable stands out as unusual. I'd like to see justification for this decision for us to discuss.